MainApp/lib/app/routes/api.js

```diff
@@ -8,21 +8,18 @@ import UserStatistics from '../models/userStatistics';
 import Authenticator from '../helpers/authentication';
 import WorkerQueue from '../background-jobs/worker-queue';
 import AdminLog from '../models/adminLog';
+import * as UUID from 'uuid-1345';
+
 
 export default function (router) {
     //http://localhost:8080/api/trialinfo
     router.post('/trialinfo', function (req, res) {
         let trialinfo = new TrialInfo();
-
-        maclib.getMac(function (err, macAddress){
-            if (err) {
-                console.log(err);
-                res.send({success: false, message: "Error getting mac address"});
-            }
-
-            let username = hash.murmurHash(macAddress);
-            trialinfo.username = username;
-
+            
+            let userIdV1=UUID.v1();// Version 1 UUID- Created using MAC address of the server and timestamp 
+            let userIdV4=UUID.v4();// Version 4 UUID- Created Using random number generator. 
```

Comment:
My only concern is that the server mac address might cause problem again. We are refactoring the username logic and are yet to add the cookie based approach. That is a logical explanation. Anyways,  I am OK with whatever the team decides. But it will be helpful to discuss it before the demo.